h2o-r/h2o-package/R/targetencoding.R

```diff
@@ -0,0 +1,104 @@
+# This file is auto-generated by h2o-3/h2o-bindings/bin/gen_R.py
```

Comment:
Not autogenerated ;)

I think this should go into an existing file. Otherwise it needs to be added to bunch of places (R package, test framework srcs)